### ISSUE TITLE
A11y_Graph Explorer_High contrast: Focus indicator for request body editor is not visible in high contrast aquatic mode.

### ISSUE THREAD

**Issue by ElinorW** [microsoftgraph/microsoft-graph-explorer-v4#2873]
**Environment Details:**
Application name: Graph Explorer 
URL:  https://developer.microsoft.com/en-us/graph/graph-explorer
Windows version: 22631.1830
Edge Version: Version 115.0.1880.3 (Official build) dev (64-bit)

**Pre-Requisites:**
Settings > Accessibility > contrast themes > Aquatic mode.

**Repro Steps:**

1. Open URL:  https://developer.microsoft.com/en-us/graph/graph-explorer
2. Tab till request body.
3. Tab till request body editor content.
4. Observe whether the focus indicator is visible or not in request body editor.

**Actual Result:**
Focus indicator for request body editor is not visible in high contrast aquatic mode, after pressing tab from request body control.

**Note: Issue is observed in desert, dusk, night sky modes also.**

**Expected Result:**
Focus indicator for request body editor should be visible in high contrast aquatic mode, after pressing tab from request body control.

**User impact:**
Low vision users will face difficulty if the focus indicator is not visible in high contrast mode.

**MAS Reference:**
[MAS 4.3.3 – User Preferences](https://aka.ms/MAS4.3.3)
